bigbluebutton-html5/package.json

```diff
@@ -5,7 +5,15 @@
     "start": "cd app;./start.sh",
     "preinstall": "npm install grunt-cli",
     "test": "wdio ./tests/webdriverio/wdio.conf.js",
-    "lint": "eslint . --ext .jsx,.js"
+    "lint": "eslint . --ext .jsx,.js",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "gitDir": "../",
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
```

Comment:
Correct me if I am wrong - if you changed a few files, but want to commit just one - this line will stage everything anyway? This may be inconvenient..
I didn't test this case.